client/parties/actions.ts

```diff
@@ -0,0 +1,121 @@
+import { PartyPayload, PartyUser } from '../../common/parties'
+import { BaseFetchFailure } from '../network/fetch-action-types'
+
+export type PartiesActions =
+  | InviteToPartyBegin
+  | InviteToPartySuccess
+  | BaseFetchFailure<'@parties/inviteToParty'>
+  | RemovePartyInviteBegin
+  | RemovePartyInviteSuccess
+  | BaseFetchFailure<'@parties/removePartyInvite'>
+  | DeclinePartyInviteBegin
+  | DeclinePartyInviteSuccess
+  | BaseFetchFailure<'@parties/declinePartyInvite'>
+  | AcceptPartyInviteBegin
+  | AcceptPartyInviteSuccess
+  | BaseFetchFailure<'@parties/acceptPartyInvite'>
+  | AddInvite
+  | RemoveInvite
+  | Init
+  | Invite
+  | Uninvite
+  | Decline
+  | Join
+  | Leave
+
+export interface InviteToPartyBegin {
+  type: '@parties/inviteToPartyBegin'
+}
+
+export interface InviteToPartySuccess {
+  type: '@parties/inviteToParty'
+  payload: void
+  error?: false
+}
+
+export interface RemovePartyInviteBegin {
+  type: '@parties/removePartyInviteBegin'
+}
+
+export interface RemovePartyInviteSuccess {
+  type: '@parties/removePartyInvite'
+  payload: void
+  error?: false
+}
+
+export interface DeclinePartyInviteBegin {
+  type: '@parties/declinePartyInviteBegin'
+}
+
+export interface DeclinePartyInviteSuccess {
+  type: '@parties/declinePartyInvite'
+  payload: void
+  error?: false
+}
+
+export interface AcceptPartyInviteBegin {
+  type: '@parties/acceptPartyInviteBegin'
+}
+
+export interface AcceptPartyInviteSuccess {
+  type: '@parties/acceptPartyInvite'
+  payload: void
+  error?: false
+}
+
+export interface AddInvite {
```

Comment:
Add and Remove feel really unnecessary? At least from the structure of these it seems like they're for when *you* have been invited to a party, but this is meant to be handled via the notification system. Unless you want some UI to display every party you're currently invited to (and I don't think we do), I can't see how these would be useful or desirable